eng/common/TestResources/sub-config/AzurePublicMsft.json

```diff
@@ -0,0 +1,11 @@
+{
+    "SubscriptionId": "2cd617ea-1866-46b1-90e3-fffb087ebf9b",
+    "TenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
+    "TestApplicationId": "f850650c-1fcf-4489-b46f-71af2e30d360",
+    "TestApplicationOid": "30511c9d-ba1a-4c7b-b422-5b543da11b3f",
+    "ProvisionerApplicationId": "f850650c-1fcf-4489-b46f-71af2e30d360",
+    "ProvisionerApplicationOid": "30511c9d-ba1a-4c7b-b422-5b543da11b3f",
+    "Environment": "AzureCloud",
+    "AzureSubscription": "Azure SDK Test Resources",
+    "EnvironmentVariables": {}
```

Comment:
Are EnvVariables still interesting here?

Reply:
They are used by other teams who have their own configs. I think the field is possibly useful in that it shows the schema of the config, but docs can do that too (since `ArmTemplateParameters` is also missing from this)

Reply:
Removed